src/konnektor/network_planners/generators/minimal_spanning_tree_network_generator.py

```diff
@@ -62,17 +62,17 @@ def __init__(
 
     def generate_ligand_network(self, components: Iterable[Component]) -> LigandNetwork:
         """
-        Generate a MST network from a list of components.
+        Generate a MST network from the given ``Component``/s.
 
         Parameters
         ----------
         components: Iterable[Component]
-        the components to be used for the LigandNetwork
+            ``Components`` to be used as nodes in the ``LigandNetwork``.
```

Comment:
Here Component is really SmallMoleculeComponent right?

Reply:
no, it could be any [gufe component](https://gufe.openfree.energy/en/latest/generated/gufe.components.html#module-gufe.components)